pkgs/development/python-modules/pythonnet/default.nix

```diff
@@ -59,36 +64,44 @@ buildPythonPackage rec {
 
     mono
 
-    NUnit371
-    UnmanagedExports127
-  ];
+  ] ++ dotnetPkgs;
 
   buildInputs = [
     glib
     mono
+  ];
+
+  checkInputs = [
+    pytest
```

Comment:
```suggestion
    pytestCheckHook
```

Reply:
Done. Works nicely, much cleaner. Thanks for the suggestion.